addons/playersuppression/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,13 @@
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+class Extended_Fired_EventHandlers {
+    class All {
+        class ADDON {
+            clientFired = QUOTE(if ((GVAR(enablePlayerSuppression) && !isDedicated)) then {_this call FUNC(fired)});
```

Comment:
`hasInterface` instead `isDedicated`

hasInterface is false on headless clients, which is probably what you want